torchmultimodal/models/clip/text_encoder.py

```diff
@@ -120,11 +127,13 @@ def forward(self, text: Tensor) -> Tensor:
 
         # [n_ctx, bs, transformer.width] -> [bs, n_ctx, transformer.width]
         embeddings = torch.permute(embeddings, (1, 0, 2))
-        embeddings = self.ln_final(embeddings)
+        hidden_state = self.ln_final(embeddings)
         # take features from the eot embedding (the highest number in each sequence)
         embeddings = self.projection(
```

Comment:
I would suggest renaming this to something like `projected_embeddings` to avoid re-using the `embeddings` name as it was used before.